[jvm-packages] maven-checkstyle-plugin:2.17 errors

## Environment info
Operating System:
Windows7 (64bit)
Compiler:
**gcc**
gcc version 4.8.1 (rev5, Built by MinGW-W64 project)
**mingw32-make**
GNU Make 3.99.90
Built for Windows32
Copyright (C) 1988-2012 Free Software Foundation, Inc.
Package used (python/R/jvm/C++):
jvm
`xgboost` version used:
0.7

I'm trying to install jvm-package for XGboost (following the official instructions http://xgboost.readthedocs.io/en/latest/build.html, http://xgboost.readthedocs.io/en/latest/jvm/#installation).
1. I  built the shared library with mingw32-make. As the result there are two files in .../xgboost/lib - libxgboost.a and libxgboost.dll.
2. Now i'm trying to built jvm-packages. In .../xgboost/jvm-packages i run mvn install and get the following:

[INFO] Scanning for projects...
[WARNING]
[WARNING] Some problems were encountered while building the effective model for ml.dmlc:xgboost4j:jar:0.7
[WARNING] 'build.plugins.plugin.version' for org.codehaus.mojo:exec-maven-plugin is missing. @ line 84, column 29
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] xgboost-jvm
[INFO] xgboost4j
[INFO] xgboost4j-spark
[INFO] xgboost4j-flink
[INFO] xgboost4j-example
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building xgboost-jvm 0.7
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ xgboost-jvm ---
[INFO] Deleting C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\target
[INFO]
[INFO] --- scalastyle-maven-plugin:0.8.0:check (checkstyle) @ xgboost-jvm ---
[WARNING] sourceDirectory is not specified or does not exist value=C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\src\main\scala
[WARNING] testSourceDirectory is not specified or does not exist value=C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\src\test\scala
Processed 0 file(s)
Found 0 errors
Found 0 warnings
Found 0 infos
Finished in 93 ms
[INFO]
[INFO] --- maven-checkstyle-plugin:2.17:check (checkstyle) @ xgboost-jvm ---
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:compile (default) @ xgboost-jvm ---
[WARNING]  Expected all dependencies to require Scala version: 2.11.8
[WARNING]  ml.dmlc:xgboost-jvm:0.7 requires scala version: 2.11.8
[WARNING]  org.scala-lang:scala-compiler:2.11.8 requires scala version: 2.11.8
[WARNING]  org.scala-lang.modules:scala-xml_2.11:1.0.4 requires scala version: 2.11.4
[WARNING] Multiple versions of scala libraries detected!
[INFO] No sources to compile
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:compile (scala-compile-first) @ xgboost-jvm ---
[WARNING]  Expected all dependencies to require Scala version: 2.11.8
[WARNING]  ml.dmlc:xgboost-jvm:0.7 requires scala version: 2.11.8
[WARNING]  org.scala-lang:scala-compiler:2.11.8 requires scala version: 2.11.8
[WARNING]  org.scala-lang.modules:scala-xml_2.11:1.0.4 requires scala version: 2.11.4
[WARNING] Multiple versions of scala libraries detected!
[INFO] No sources to compile
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:add-source (scala-compile-first) @ xgboost-jvm ---
[INFO] Add Source directory: C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\src\main\scala
[INFO] Add Test Source directory: C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\src\test\scala
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:compile (compile) @ xgboost-jvm ---
[WARNING]  Expected all dependencies to require Scala version: 2.11.8
[WARNING]  ml.dmlc:xgboost-jvm:0.7 requires scala version: 2.11.8
[WARNING]  org.scala-lang:scala-compiler:2.11.8 requires scala version: 2.11.8
[WARNING]  org.scala-lang.modules:scala-xml_2.11:1.0.4 requires scala version: 2.11.4
[WARNING] Multiple versions of scala libraries detected!
[INFO] No sources to compile
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:testCompile (test-compile) @ xgboost-jvm ---
[WARNING]  Expected all dependencies to require Scala version: 2.11.8
[WARNING]  ml.dmlc:xgboost-jvm:0.7 requires scala version: 2.11.8
[WARNING]  org.scala-lang:scala-compiler:2.11.8 requires scala version: 2.11.8
[WARNING]  org.scala-lang.modules:scala-xml_2.11:1.0.4 requires scala version: 2.11.4
[WARNING] Multiple versions of scala libraries detected!
[INFO] No sources to compile
[INFO]
[INFO] --- scalatest-maven-plugin:1.0:test (test) @ xgboost-jvm ---
Discovery starting.
Discovery completed in 140 milliseconds.
Run starting. Expected test count is: 0
DiscoverySuite:
Run completed in 187 milliseconds.
Total number of tests run: 0
Suites: completed 1, aborted 0
Tests: succeeded 0, failed 0, canceled 0, ignored 0, pending 0
No tests were executed.
[INFO]
[INFO] --- maven-assembly-plugin:2.6:single (make-assembly) @ xgboost-jvm ---
[INFO] Assemblies have been skipped per configuration of the skipAssembly parameter.
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building xgboost4j 0.7
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ xgboost4j ---
[INFO] Deleting C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\target
[INFO]
[INFO] --- scalastyle-maven-plugin:0.8.0:check (checkstyle) @ xgboost4j ---
Processed 13 file(s)
Found 0 errors
Found 0 warnings
Found 0 infos
Finished in 1092 ms
[INFO]
[INFO] --- maven-checkstyle-plugin:2.17:check (checkstyle) @ xgboost4j ---
[INFO] There are 82 errors reported by Checkstyle 6.11.2 with checkstyle.xml ruleset.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[2] (regexp) RegexpSingleline: No trailing whitespace allowed.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[7] (regexp) RegexpSingleline: No trailing whitespace allowed.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[23] (sizes) LineLength: Line is longer than 100 characters (found 104).
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[28,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[28,37] (naming) MethodName: Method name 'XGBGetLastError' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[30,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[30,34] (naming) MethodName: Method name 'XGDMatrixCreateFromFile' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[32,9] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[32,27] (naming) MethodName: Method name 'XGDMatrixCreateFromDataIter' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[35] (sizes) LineLength: Line is longer than 100 characters (found 101).
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[35,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[35,34] (naming) MethodName: Method name 'XGDMatrixCreateFromCSREx' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[38] (sizes) LineLength: Line is longer than 100 characters (found 101).
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[38,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[38,34] (naming) MethodName: Method name 'XGDMatrixCreateFromCSCEx' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[41,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[41,34] (naming) MethodName: Method name 'XGDMatrixCreateFromMat' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[44,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[44,34] (naming) MethodName: Method name 'XGDMatrixSliceDMatrix' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[46,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[46,34] (naming) MethodName: Method name 'XGDMatrixFree' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[48,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[48,34] (naming) MethodName: Method name 'XGDMatrixSaveBinary' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[50,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[50,34] (naming) MethodName: Method name 'XGDMatrixSetFloatInfo' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[52,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[52,34] (naming) MethodName: Method name 'XGDMatrixSetUIntInfo' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[54,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[54,34] (naming) MethodName: Method name 'XGDMatrixSetGroup' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[56,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[56,34] (naming) MethodName: Method name 'XGDMatrixGetFloatInfo' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[58,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[58,34] (naming) MethodName: Method name 'XGDMatrixGetUIntInfo' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[60,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[60,34] (naming) MethodName: Method name 'XGDMatrixNumRow' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[62,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[62,34] (naming) MethodName: Method name 'XGBoosterCreate' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[64,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[64,34] (naming) MethodName: Method name 'XGBoosterFree' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[66,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[66,34] (naming) MethodName: Method name 'XGBoosterSetParam' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[68,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[68,34] (naming) MethodName: Method name 'XGBoosterUpdateOneIter' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[70,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[70,34] (naming) MethodName: Method name 'XGBoosterBoostOneIter' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[73,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[73,34] (naming) MethodName: Method name 'XGBoosterEvalOneIter' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[76,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[76,34] (naming) MethodName: Method name 'XGBoosterPredict' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[79,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[79,34] (naming) MethodName: Method name 'XGBoosterLoadModel' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[81,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[81,34] (naming) MethodName: Method name 'XGBoosterSaveModel' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[83,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[83,34] (naming) MethodName: Method name 'XGBoosterLoadModelFromBuffer' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[85,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[85,34] (naming) MethodName: Method name 'XGBoosterGetModelRaw' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[87,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[87,34] (naming) MethodName: Method name 'XGBoosterDumpModelEx' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[90,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[90,34] (naming) MethodName: Method name 'XGBoosterGetAttr' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[91,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[91,34] (naming) MethodName: Method name 'XGBoosterSetAttr' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[92,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[92,34] (naming) MethodName: Method name 'XGBoosterLoadRabitCheckpoint' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[93,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[93,34] (naming) MethodName: Method name 'XGBoosterSaveRabitCheckpoint' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[96,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[96,34] (naming) MethodName: Method name 'RabitInit' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[97,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[97,34] (naming) MethodName: Method name 'RabitFinalize' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[98,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[98,34] (naming) MethodName: Method name 'RabitTrackerPrint' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[99,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[99,34] (naming) MethodName: Method name 'RabitGetRank' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[100,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[100,34] (naming) MethodName: Method name 'RabitGetWorldSize' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[101,16] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[101,34] (naming) MethodName: Method name 'RabitVersionNumber' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[105,9] (modifier) ModifierOrder: 'static' modifier out of order with the JLS suggestions.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[105,27] (naming) MethodName: Method name 'RabitAllreduce' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
[ERROR] src\main\java\ml\dmlc\xgboost4j\java\XGBoostJNI.java:[107] (regexp) RegexpSingleline: No trailing whitespace allowed.
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] xgboost-jvm ........................................ SUCCESS [  5.225 s]
[INFO] xgboost4j .......................................... FAILURE [  1.872 s]
[INFO] xgboost4j-spark .................................... SKIPPED
[INFO] xgboost4j-flink .................................... SKIPPED
[INFO] xgboost4j-example .................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 7.284 s
[INFO] Finished at: 2017-05-25T12:39:26+03:00
[INFO] Final Memory: 21M/245M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (checkstyle) on project xgboost4j: You have 82 Checkstyle violations. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :xgboost4j

and I typed Enter and get this problem:

C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j>exit
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ xgboost4j ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\src\main\resources
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:compile (default) @ xgboost4j ---
[WARNING]  Expected all dependencies to require Scala version: 2.11.8
[WARNING]  com.typesafe.akka:akka-actor_2.11:2.3.11 requires scala version: 2.11.5
[WARNING] Multiple versions of scala libraries detected!
[INFO] C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\src\main\java:-1: info: compiling
[INFO] C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\src\main\scala:-1: info: compiling
[INFO] Compiling 24 source files to C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\target\classes at 1495728077878
[WARNING] warning: there was one deprecation warning; re-run with -deprecation for details
[WARNING] warning: there were four feature warnings; re-run with -feature for details
[WARNING] two warnings found
[INFO] prepare-compile in 0 s
[INFO] compile in 17 s
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:compile (scala-compile-first) @ xgboost4j ---
[WARNING]  Expected all dependencies to require Scala version: 2.11.8
[WARNING]  com.typesafe.akka:akka-actor_2.11:2.3.11 requires scala version: 2.11.5
[WARNING] Multiple versions of scala libraries detected!
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:add-source (scala-compile-first) @ xgboost4j ---
[INFO] Add Source directory: C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\src\main\scala
[INFO] Add Test Source directory: C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\src\test\scala
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ xgboost4j ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 14 source files to C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\target\classes
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:compile (compile) @ xgboost4j ---
[WARNING]  Expected all dependencies to require Scala version: 2.11.8
[WARNING]  com.typesafe.akka:akka-actor_2.11:2.3.11 requires scala version: 2.11.5
[WARNING] Multiple versions of scala libraries detected!
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ xgboost4j ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\src\test\resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ xgboost4j ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 2 source files to C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\target\test-classes
[WARNING] /C:/Users/ilya.rabotaev/Documents/tools/xgboost/jvm-packages/xgboost4j/src/test/java/ml/dmlc/xgboost4j/java/DMatrixTest.java: C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\src\test\java\ml\dmlc\xgboost4j\java\DMatrixTest.java uses or overrides a deprecated API.
[WARNING] /C:/Users/ilya.rabotaev/Documents/tools/xgboost/jvm-packages/xgboost4j/src/test/java/ml/dmlc/xgboost4j/java/DMatrixTest.java: Recompile with -Xlint:deprecation for details.
[INFO]
[INFO] --- scala-maven-plugin:3.2.2:testCompile (test-compile) @ xgboost4j ---
[WARNING]  Expected all dependencies to require Scala version: 2.11.8
[WARNING]  com.typesafe.akka:akka-actor_2.11:2.3.11 requires scala version: 2.11.5
[WARNING] Multiple versions of scala libraries detected!
[INFO] C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\src\test\java:-1: info: compiling
[INFO] C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\src\test\scala:-1: info: compiling
[INFO] Compiling 5 source files to C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\target\test-classes at 1495728098613
[WARNING] warning: there were two deprecation warnings; re-run with -deprecation for details
[WARNING] one warning found
[INFO] prepare-compile in 0 s
[INFO] compile in 11 s
[INFO]
[INFO] --- maven-surefire-plugin:2.19.1:test (default-test) @ xgboost4j ---

-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running ml.dmlc.xgboost4j.java.BoosterImplTest
▒▒▒ 25, 2017 7:01:51 PM ml.dmlc.xgboost4j.java.NativeLibLoader smartLoad
SEVERE: failed to load library from both native path and jar
▒▒▒ 25, 2017 7:01:51 PM ml.dmlc.xgboost4j.java.DMatrix <clinit>
SEVERE: load native library failed.
▒▒▒ 25, 2017 7:01:51 PM ml.dmlc.xgboost4j.java.DMatrix <clinit>
SEVERE: java.io.FileNotFoundException: File /lib/xgboost4j.dll was not found inside JAR.
Tests run: 10, Failures: 0, Errors: 10, Skipped: 0, Time elapsed: 0.218 sec <<< FAILURE! - in ml.dmlc.xgboost4j.java.BoosterImplTest
testFastHistoDepthwiseMaxDepth(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0.156 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.testFastHistoDepthwiseMaxDepth(BoosterImplTest.java:246)

testFastHistoLossGuideMaxBin(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.testFastHistoLossGuideMaxBin(BoosterImplTest.java:214)

saveLoadModelWithPath(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.saveLoadModelWithPath(BoosterImplTest.java:118)

testCV(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.testCV(BoosterImplTest.java:295)

testBoosterBasic(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.testBoosterBasic(BoosterImplTest.java:102)

testFastHistoDepthwiseMaxDepthMaxBin(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.testFastHistoDepthwiseMaxDepthMaxBin(BoosterImplTest.java:267)

testFastHistoDepthWise(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.testFastHistoDepthWise(BoosterImplTest.java:171)

testFastHistoLossGuide(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.testFastHistoLossGuide(BoosterImplTest.java:192)

testDumpModelJson(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.testDumpModelJson(BoosterImplTest.java:236)

saveLoadModelWithStream(ml.dmlc.xgboost4j.java.BoosterImplTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.BoosterImplTest.saveLoadModelWithStream(BoosterImplTest.java:136)

Running ml.dmlc.xgboost4j.java.DMatrixTest
Tests run: 8, Failures: 0, Errors: 8, Skipped: 0, Time elapsed: 0 sec <<< FAILURE! - in ml.dmlc.xgboost4j.java.DMatrixTest
testCreateFromCSCEx(ml.dmlc.xgboost4j.java.DMatrixTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromCSCEx([J[I[FI[J)I
        at ml.dmlc.xgboost4j.java.DMatrixTest.testCreateFromCSCEx(DMatrixTest.java:155)

testCreateFromCSREx(ml.dmlc.xgboost4j.java.DMatrixTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromCSREx([J[I[FI[J)I
        at ml.dmlc.xgboost4j.java.DMatrixTest.testCreateFromCSREx(DMatrixTest.java:106)

testCreateFromFile(ml.dmlc.xgboost4j.java.DMatrixTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromFile(Ljava/lang/String;I[J)I
        at ml.dmlc.xgboost4j.java.DMatrixTest.testCreateFromFile(DMatrixTest.java:60)

testCreateFromDataIterator(ml.dmlc.xgboost4j.java.DMatrixTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromDataIter(Ljava/util/Iterator;Ljava/lang/String;[J)I
        at ml.dmlc.xgboost4j.java.DMatrixTest.testCreateFromDataIterator(DMatrixTest.java:49)

testCreateFromDenseMatrix(ml.dmlc.xgboost4j.java.DMatrixTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromMat([FIIF[J)I
        at ml.dmlc.xgboost4j.java.DMatrixTest.testCreateFromDenseMatrix(DMatrixTest.java:184)

testCreateFromCSC(ml.dmlc.xgboost4j.java.DMatrixTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromCSCEx([J[I[FI[J)I
        at ml.dmlc.xgboost4j.java.DMatrixTest.testCreateFromCSC(DMatrixTest.java:130)

testCreateFromCSR(ml.dmlc.xgboost4j.java.DMatrixTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromCSREx([J[I[FI[J)I
        at ml.dmlc.xgboost4j.java.DMatrixTest.testCreateFromCSR(DMatrixTest.java:84)

testCreateFromDenseMatrixWithMissingValue(ml.dmlc.xgboost4j.java.DMatrixTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.UnsatisfiedLinkError: ml.dmlc.xgboost4j.java.XGBoostJNI.XGDMatrixCreateFromMat([FIIF[J)I
        at ml.dmlc.xgboost4j.java.DMatrixTest.testCreateFromDenseMatrixWithMissingValue(DMatrixTest.java:223)

Running ml.dmlc.xgboost4j.scala.rabit.RabitTrackerConnectionHandlerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.045 sec - in ml.dmlc.xgboost4j.scala.rabit.RabitTrackerConnectionHandlerTest

Results :

Tests in error:
  BoosterImplTest.saveLoadModelWithPath:118 ▒ UnsatisfiedLink ml.dmlc.xgboost4j....
  BoosterImplTest.saveLoadModelWithStream:136 ▒ UnsatisfiedLink ml.dmlc.xgboost4...
  BoosterImplTest.testBoosterBasic:102 ▒ UnsatisfiedLink ml.dmlc.xgboost4j.java....
  BoosterImplTest.testCV:295 ▒ UnsatisfiedLink ml.dmlc.xgboost4j.java.XGBoostJNI...
  BoosterImplTest.testDumpModelJson:236 ▒ UnsatisfiedLink ml.dmlc.xgboost4j.java...
  BoosterImplTest.testFastHistoDepthWise:171 ▒ UnsatisfiedLink ml.dmlc.xgboost4j...
  BoosterImplTest.testFastHistoDepthwiseMaxDepth:246 ▒ UnsatisfiedLink ml.dmlc.x...
  BoosterImplTest.testFastHistoDepthwiseMaxDepthMaxBin:267 ▒ UnsatisfiedLink ml....
  BoosterImplTest.testFastHistoLossGuide:192 ▒ UnsatisfiedLink ml.dmlc.xgboost4j...
  BoosterImplTest.testFastHistoLossGuideMaxBin:214 ▒ UnsatisfiedLink ml.dmlc.xgb...
  DMatrixTest.testCreateFromCSC:130 ▒ UnsatisfiedLink ml.dmlc.xgboost4j.java.XGB...
  DMatrixTest.testCreateFromCSCEx:155 ▒ UnsatisfiedLink ml.dmlc.xgboost4j.java.X...
  DMatrixTest.testCreateFromCSR:84 ▒ UnsatisfiedLink ml.dmlc.xgboost4j.java.XGBo...
  DMatrixTest.testCreateFromCSREx:106 ▒ UnsatisfiedLink ml.dmlc.xgboost4j.java.X...
  DMatrixTest.testCreateFromDataIterator:49 ▒ UnsatisfiedLink ml.dmlc.xgboost4j....
  DMatrixTest.testCreateFromDenseMatrix:184 ▒ UnsatisfiedLink ml.dmlc.xgboost4j....
  DMatrixTest.testCreateFromDenseMatrixWithMissingValue:223 ▒ UnsatisfiedLink ml...
  DMatrixTest.testCreateFromFile:60 ▒ UnsatisfiedLink ml.dmlc.xgboost4j.java.XGB...

Tests run: 22, Failures: 0, Errors: 18, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] xgboost-jvm ........................................ SUCCESS [  4.138 s]
[INFO] xgboost4j .......................................... FAILURE [03:38 min]
[INFO] xgboost4j-spark .................................... SKIPPED
[INFO] xgboost4j-flink .................................... SKIPPED
[INFO] xgboost4j-example .................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 03:43 min
[INFO] Finished at: 2017-05-25T19:01:53+03:00
[INFO] Final Memory: 29M/318M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project xgboost4j: There are test failures.
[ERROR]
[ERROR] Please refer to C:\Users\ilya.rabotaev\Documents\tools\xgboost\jvm-packages\xgboost4j\target\surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :xgboost4j

--- a/jvm-packages/checkstyle-suppressions.xml
+++ b/jvm-packages/checkstyle-suppressions.xml
@@ -28,6 +28,5 @@
 -->
 
 <suppressions>
-<suppress checks=".*"
-files="xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/XGBoostJNI.java"/>
+  <suppress checks=".*" files="XGBoostJNI.java"/>
 </suppressions>

